api/handlers_e2e_test.go

```diff
@@ -2129,3 +2129,447 @@ func runBoxCreateMutateDelete(t *testing.T, comparator boxTestComparator) {
 func TestBoxCreateMutateDeleteAgainstHandler(t *testing.T) {
 	runBoxCreateMutateDelete(t, compareAppBoxesAgainstHandler)
 }
+
+func makeRequest(t *testing.T, listenAddr string, path string, includeDeleted bool) (*http.Response, []byte) {
+	// make a real HTTP request
+	t.Log("making HTTP request path", path)
+	req := fmt.Sprintf("http://%s%s?pretty", listenAddr, path)
+	if includeDeleted {
+		req = fmt.Sprintf("%s&include-all=true", req)
+	}
+	resp, err := http.Get(req)
+	require.NoError(t, err)
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+	require.NoError(t, err)
+	return resp, body
+}
+
+// recreate tests from  common.sh/create_delete_tests()
```

Comment:
these tests are replicating those in common.sh/create_delete_tests(). Given how tedious it is to create the vb files, not all tests are replicated. Let me know if you see cases that should be covered.